### ISSUE TITLE
Undo tweaks not working

### ISSUE THREAD

**Issue by NichirinSword** [ChrisTitusTech/winutil#905]
Undo tweak options not working. I tried to undo some options but didn't work. In windows update, location setting and edge browser shows that some settings are managed by your organization.

**Comment by ChrisTitusTech** [comment]
Working on a fix #377 

In the meantime you can do this

--How to Remove Managed by your organization from Microsoft Edge--

Launch the Windows Registry Editor. Press Win + R on the keyboard and type regedit in the Run box.
Go to the following key: HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft.
Locate the Edge folder and right-click it.
Select Delete from the menu.
Alternatively, you can delete all the parameters inside this folder.
Now, open the following key: HKEY_CURRENT_USER\SOFTWARE\Policies\Microsoft\Edge
Delete its contents as well.
Restart the browser. The message should be gone now.

**Comment by abhinandanangra** [comment]
I tried using the same settings mentioned above, but the message is still there.
I have used the Winutil utility and did the recommended tweak for laptops, adding the following.

1.  Disable Hibernation
2.  Disable mouse acceleration
3.  Changing DNS to Cloudflare Malware

I also tried to revert back to the system restore point, which the utility created before all the tweaks and software installations, I did.
Neither the software(VS Code) is removed, nor is the message in Edge.
